python/graphstorm/gconstruct/utils.py

```diff
@@ -167,6 +167,7 @@ def worker_fn(worker_id, task_queue, res_queue, user_parser):
         while True:
             # If the queue is empty, it will raise the Empty exception.
             i, in_file = task_queue.get_nowait()
+            logging.debug("%d Processing %s", worker_id, in_file)
```

Comment:
Can we control the different log level to avoid output more information?

Reply:
logging.debug will not print until you use --logging-level debug.